### ISSUE TITLE
simulator.exe cannot be opened

### ISSUE THREAD

**Issue by tk-s05** [lgsvl/simulator#1379]
Hi,
I can't launch the simulator.
I get an error when I try to start the simulator.　
・I uninstalled the simulator but couldn't start it
・I can start it by logging in to the PC with another account
![1](https://user-images.githubusercontent.com/82795532/115184937-cf5a7b80-a119-11eb-8b3d-652d978943ec.png)
 Hope for your help, thanks.

**Comment by EricBoiseLGSVL** [comment]
Please post OS, version of simulator, gpu and then post player.log so we can see the error.

**Comment by tk-s05** [comment]
OS:Win10, GPU:Quadro P6000, version:2021.1 release

I downloaded from the SVL Simulator homepage and unzipped the file.
I used to be able to start it, but suddenly I can't start it.
Is there any other information needed to resolve this issue?

＜windows log＞
------------------------------------------------------------------------------------------------------------------------
Log name: Application
Source: Application Error
Date: 2021/04/21 13:43:49
Event ID: 1000
Task category: (100)
Level: Error
Keywords: classic
User: N / A
Computer:　（Keep it private）
Description:
Failed application name: simulator.exe, version: 2019.4.18.4260, timestamp: 0x5ffdc692
Failed module name: UnityPlayer.dll, version: 2019.4.18.4260, timestamp: 0x5ffdc82f
Exception code: 0xc0000005
Fault offset: 0x00000000009234c4
Failing process ID: 0xed90
Failed application start time: 0x01d73668bde2d55c
Failed application path: C: \ Users \ lgsvl \ Desktop \ svlsimulator-windows64-2021.1 \ simulator.exe
Failed module path: C: \ Users \ lgsvl \ Desktop \ svlsimulator-windows64-2021.1 \ UnityPlayer.dll
Report ID: caf10a98-c798-4176-8530-db7f067e7057
The full name of the failing package:
Application ID associated with the failing package:

**Comment by tk-s05** [comment]
player.log
-----------------------------------------------------------------------------------------------------------
Mono path[0] = 'C:/Users/lgsvl/Desktop/svlsimulator-windows64-2021.1/simulator_Data/Managed'
Mono config path = 'C:/Users/lgsvl/Desktop/svlsimulator-windows64-2021.1/MonoBleedingEdge/etc'
Loading player data from C:/Users/lgsvl/Desktop/svlsimulator-windows64-2021.1/simulator_Data/data.unity3d
Initialize engine version: 2019.4.18f1 (3310a4d4f880)
[Subsystems] Discovering subsystems at path C:/Users/lgsvl/Desktop/svlsimulator-windows64-2021.1/simulator_Data/UnitySubsystems
GfxDevice: creating device client; threaded=1
Direct3D:
    Version:  Direct3D 11.0 [level 11.1]
    Renderer: NVIDIA Quadro P6000 (ID=0x1b30)
    Vendor:   
    VRAM:     24431 MB
    Driver:   26.21.14.3170
Begin MonoManager ReloadAssembly
- Completed reload, in  0.118 seconds
WARNING: Shader Unsupported: 'HDRP/Lit' - All passes removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
D3D11 device created for Microsoft Media Foundation video decoding.
<RI> Initializing input.

New input system (experimental) initialized
XInput1_3.dll not found. Trying XInput9_1_0.dll instead...
<RI> Input initialized.

<RI> Initialized touch support.

WARNING: Shader Unsupported: 'Shader Graphs/CloudGenerator' - All passes removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
WARNING: Shader Unsupported: 'Shader Graphs/ParticleLitSoft' - All passes removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
UnloadTime: 1.048307 ms
WARNING: Shader Unsupported: 'Shader Graphs/VehicleLightsShader' - All passes removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
found manifest at Controllables\controllable_ChargingStation\manifest.json 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Fallback handler could not load library C:/Users/lgsvl/Desktop/svlsimulator-windows64-2021.1/simulator_Data/Mono/data-0000015729E2F340.dll
WARNING: Shader Unsupported: 'HDRP/Lit' - All passes removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
found manifest at Controllables\controllable_TrafficCone\manifest.json 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Fallback handler could not load library C:/Users/lgsvl/Desktop/svlsimulator-windows64-2021.1/simulator_Data/Mono/data-00000157517BE030.dll
WARNING: Shader Unsupported: 'HDRP/Lit' - All passes removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
found manifest at Controllables\controllable_TriggerZone\manifest.json 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Fallback handler could not load library C:/Users/lgsvl/Desktop/svlsimulator-windows64-2021.1/simulator_Data/Mono/data-00000157517C17E0.dll
WARNING: Shader Unsupported: 'HDRP/Unlit' - All passes removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
Loaded 3 NPCs behaviours and 6 NPC models in 00:00:00.2087753 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Current Database Version: 7 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

Final Database Version: 7 
(Filename: C:\buildslave\unity\build\Runtime/Export/Debug/Debug.bindings.h Line: 35)

[CONN] Instance URL https://wise.svlsimulator.com/
[CONN] Connecting to WISE
Setting up 12 worker threads for Enlighten.
  Thread -> id: 9c90 -> priority: 1 
  Thread -> id: 96b8 -> priority: 1 
  Thread -> id: 78d4 -> priority: 1 
  Thread -> id: c330 -> priority: 1 
  Thread -> id: 1118 -> priority: 1 
  Thread -> id: c134 -> priority: 1 
  Thread -> id: 557c -> priority: 1 
  Thread -> id: a864 -> priority: 1 
  Thread -> id: 12da4 -> priority: 1 
  Thread -> id: 125d8 -> priority: 1 
  Thread -> id: 146fc -> priority: 1 
  Thread -> id: 8fa0 -> priority: 1 
Crash!!!
SymInit: Symbol-SearchPath: '.;C:\Users\lgsvl\Desktop\svlsimulator-windows64-2021.1;C:\Users\lgsvl\Desktop\svlsimulator-windows64-2021.1;C:\Windows;C:\Windows\system32;SRV*C:\websymbols*http://msdl.microsoft.com/download/symbols;', symOptions: 534, UserName: 'lgsvl'
OS-Version: 10.0.0
C:\Users\lgsvl\Desktop\svlsimulator-windows64-2021.1\simulator.exe:simulator.exe (00007FF75DB30000), size: 667648 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 2019.4.18.4260
C:\Windows\SYSTEM32\ntdll.dll:ntdll.dll (00007FFE32000000), size: 1896448 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4225
C:\Windows\System32\KERNEL32.DLL:KERNEL32.DLL (00007FFE2F670000), size: 704512 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.3630
C:\Windows\System32\KERNELBASE.dll:KERNELBASE.dll (00007FFE2F1E0000), size: 2220032 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.3986
C:\Windows\System32\SYSFER.DLL:SYSFER.DLL (0000000050C40000), size: 602112 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 14.2.3329.1000
C:\Users\lgsvl\Desktop\svlsimulator-windows64-2021.1\UnityPlayer.dll:UnityPlayer.dll (00007FFDD6B00000), size: 26882048 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 2019.4.18.4260
C:\Windows\System32\USER32.dll:USER32.dll (00007FFE31D10000), size: 1466368 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\win32u.dll:win32u.dll (00007FFE2F570000), size: 122880 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\GDI32.dll:GDI32.dll (00007FFE31E80000), size: 212992 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\gdi32full.dll:gdi32full.dll (00007FFE2EC80000), size: 1585152 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\ole32.dll:ole32.dll (00007FFE2F880000), size: 1282048 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\combase.dll:combase.dll (00007FFE31890000), size: 2908160 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4225
C:\Windows\SYSTEM32\VERSION.dll:VERSION.dll (00007FFE2B1D0000), size: 40960 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\ucrtbase.dll:ucrtbase.dll (00007FFE2F400000), size: 999424 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.3659
C:\Windows\System32\RPCRT4.dll:RPCRT4.dll (00007FFE31BE0000), size: 1183744 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\bcryptPrimitives.dll:bcryptPrimitives.dll (00007FFE2F500000), size: 438272 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4046
C:\Windows\System32\msvcrt.dll:msvcrt.dll (00007FFE2F9C0000), size: 647168 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 7.0.14393.2457
C:\Windows\System32\sechost.dll:sechost.dll (00007FFE2F720000), size: 364544 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.3808
C:\Windows\System32\SHLWAPI.dll:SHLWAPI.dll (00007FFE31FA0000), size: 335872 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\SETUPAPI.dll:SETUPAPI.dll (00007FFE31110000), size: 4362240 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2608
C:\Windows\System32\cfgmgr32.dll:cfgmgr32.dll (00007FFE2F0E0000), size: 270336 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\ADVAPI32.dll:ADVAPI32.dll (00007FFE31540000), size: 663552 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2969
C:\Windows\System32\SHELL32.dll:SHELL32.dll (00007FFE2FC00000), size: 22040576 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4225
C:\Windows\System32\windows.storage.dll:windows.storage.dll (00007FFE2E5A0000), size: 7155712 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\powrprof.dll:powrprof.dll (00007FFE2E490000), size: 311296 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\kernel.appcore.dll:kernel.appcore.dll (00007FFE2E480000), size: 61440 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2312
C:\Windows\System32\shcore.dll:shcore.dll (00007FFE2E4F0000), size: 688128 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\profapi.dll:profapi.dll (00007FFE2E460000), size: 81920 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\OLEAUT32.dll:OLEAUT32.dll (00007FFE31EE0000), size: 786432 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4225
C:\Windows\System32\msvcp_win.dll:msvcp_win.dll (00007FFE2EFE0000), size: 638976 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2999
C:\Windows\SYSTEM32\OPENGL32.dll:OPENGL32.dll (00007FFDD69D0000), size: 1191936 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\SYSTEM32\WINMM.dll:WINMM.dll (00007FFE2B890000), size: 143360 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\IMM32.dll:IMM32.dll (00007FFE2F590000), size: 188416 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\CRYPT32.dll:CRYPT32.dll (00007FFE2EE10000), size: 1871872 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\MSASN1.dll:MSASN1.dll (00007FFE2E4E0000), size: 65536 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\WS2_32.dll:WS2_32.dll (00007FFE317B0000), size: 434176 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.3241
C:\Windows\SYSTEM32\bcrypt.dll:bcrypt.dll (00007FFE2E040000), size: 176128 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4046
C:\Windows\SYSTEM32\HID.DLL:HID.DLL (00007FFE2D420000), size: 49152 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\SYSTEM32\WINHTTP.dll:WINHTTP.dll (00007FFE2A080000), size: 839680 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\SYSTEM32\WINMMBASE.dll:WINMMBASE.dll (00007FFE2B240000), size: 176128 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\SYSTEM32\GLU32.dll:GLU32.dll (00007FFE20A10000), size: 184320 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\SYSTEM32\DDRAW.dll:DDRAW.dll (00007FFDD68D0000), size: 1011712 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.953
C:\Windows\SYSTEM32\DCIMAN32.dll:DCIMAN32.dll (00007FFE2A2F0000), size: 32768 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\psapi.dll:psapi.dll (00007FFE31BD0000), size: 32768 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\system32\uxtheme.dll:uxtheme.dll (00007FFE2CB60000), size: 610304 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Users\lgsvl\Desktop\svlsimulator-windows64-2021.1\MonoBleedingEdge\EmbedRuntime\mono-2.0-bdwgc.dll:mono-2.0-bdwgc.dll (00007FFDD5560000), size: 7786496 (result: 0), SymType: '-deferred-', PDB: ''
C:\Windows\system32\mswsock.dll:mswsock.dll (00007FFE2DD90000), size: 380928 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.3659
C:\Windows\System32\MSCTF.dll:MSCTF.dll (00007FFE31650000), size: 1421312 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4225
C:\Windows\system32\dwmapi.dll:dwmapi.dll (00007FFE2B1E0000), size: 155648 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\SYSTEM32\d3d11.dll:d3d11.dll (00007FFE26420000), size: 2842624 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\SYSTEM32\dxgi.dll:dxgi.dll (00007FFE2D4C0000), size: 651264 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2457
C:\Windows\System32\DriverStore\FileRepository\nvwu.inf_amd64_7626307a8d01b777\nvldumdx.dll:nvldumdx.dll (00007FFE289D0000), size: 974848 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 26.21.14.3170
C:\Windows\System32\WINTRUST.DLL:WINTRUST.DLL (00007FFE2F080000), size: 348160 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4225
C:\Windows\System32\imagehlp.dll:imagehlp.dll (00007FFE31EC0000), size: 114688 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\SYSTEM32\CRYPTSP.dll:CRYPTSP.dll (00007FFE2DF30000), size: 94208 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2969
C:\Windows\system32\rsaenh.dll:rsaenh.dll (00007FFE2D960000), size: 208896 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2969
C:\Windows\SYSTEM32\CRYPTBASE.dll:CRYPTBASE.dll (00007FFE2DF50000), size: 45056 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\DriverStore\FileRepository\nvwu.inf_amd64_7626307a8d01b777\nvwgf2umx.dll:nvwgf2umx.dll (00007FFDD2A70000), size: 39546880 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 26.21.14.3170
C:\Windows\System32\clbcatq.dll:clbcatq.dll (00007FFE2F5D0000), size: 651264 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 2001.12.10941.16384
C:\Windows\system32\wbem\wbemprox.dll:wbemprox.dll (00007FFE26170000), size: 65536 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\SYSTEM32\wbemcomn.dll:wbemcomn.dll (00007FFE27750000), size: 520192 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\system32\wbem\wbemsvc.dll:wbemsvc.dll (00007FFE25940000), size: 81920 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\system32\wbem\fastprox.dll:fastprox.dll (00007FFE259E0000), size: 999424 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\MMDevApi.dll:MMDevApi.dll (00007FFE26B80000), size: 454656 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\DEVOBJ.dll:DEVOBJ.dll (00007FFE2D140000), size: 163840 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\PROPSYS.dll:PROPSYS.dll (00007FFE29EF0000), size: 1593344 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 7.0.14393.4169
C:\Windows\SYSTEM32\AUDIOSES.DLL:AUDIOSES.DLL (00007FFE28550000), size: 606208 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\SYSTEM32\wintypes.dll:wintypes.dll (00007FFE2AA40000), size: 1245184 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4225
C:\Windows\SYSTEM32\xinput1_4.dll:xinput1_4.dll (00007FFE29810000), size: 57344 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\SYSTEM32\xinput9_1_0.dll:xinput9_1_0.dll (00007FFE297F0000), size: 28672 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\SYSTEM32\dcomp.dll:dcomp.dll (00007FFE2C4C0000), size: 1380352 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Users\lgsvl\Desktop\svlsimulator-windows64-2021.1\simulator_Data\Plugins\x86_64\sqlite3.dll:sqlite3.dll (0000000180000000), size: 1933312 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 3.29.0.0
C:\Windows\SYSTEM32\ntmarta.dll:ntmarta.dll (00007FFE2D170000), size: 204800 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\IME\IMEJP\imjptip.dll:imjptip.dll (00007FFE23420000), size: 2875392 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2608
C:\Windows\SYSTEM32\iertutil.dll:iertutil.dll (00007FFE27E70000), size: 2826240 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 11.0.14393.4225
C:\Windows\SYSTEM32\OLEACC.dll:OLEACC.dll (00007FFE29B80000), size: 413696 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 7.2.14393.4169
C:\Windows\System32\IME\IMEJP\imjpapi.dll:imjpapi.dll (00007FFE22800000), size: 2195456 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\SYSTEM32\ncrypt.dll:ncrypt.dll (00007FFE2E010000), size: 155648 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4046
C:\Windows\SYSTEM32\NTASN1.dll:NTASN1.dll (00007FFE2DFD0000), size: 241664 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\IME\shared\imjkapi.dll:imjkapi.dll (00007FFE28890000), size: 212992 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\SYSTEM32\policymanager.dll:policymanager.dll (00007FFE28240000), size: 274432 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\SYSTEM32\msvcp110_win.dll:msvcp110_win.dll (00007FFE2CE90000), size: 593920 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2007
C:\Windows\System32\IME\IMEJP\imjppred.dll:imjppred.dll (00007FFE252F0000), size: 888832 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\IME\shared\imetip.dll:imetip.dll (00007FFE21420000), size: 1400832 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\SYSTEM32\DUI70.dll:DUI70.dll (00007FFE20BD0000), size: 1736704 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\IME\shared\imecfm.dll:imecfm.dll (00007FFE283C0000), size: 208896 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2457
C:\Windows\SYSTEM32\wer.dll:wer.dll (00007FFE29E40000), size: 716800 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\System32\IME\SHARED\imesearchdll.dll:imesearchdll.dll (00007FFE29650000), size: 126976 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\SYSTEM32\iphlpapi.DLL:iphlpapi.DLL (00007FFE2DAD0000), size: 229376 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2339
C:\Windows\System32\NSI.dll:NSI.dll (00007FFE2F5C0000), size: 32768 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.3297
C:\Windows\SYSTEM32\dhcpcsvc6.DLL:dhcpcsvc6.DLL (00007FFE2BA40000), size: 90112 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.3930
C:\Windows\SYSTEM32\dhcpcsvc.DLL:dhcpcsvc.DLL (00007FFE2BA20000), size: 106496 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.3930
C:\Windows\SYSTEM32\DNSAPI.dll:DNSAPI.dll (00007FFE2DB20000), size: 663552 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.4169
C:\Windows\SYSTEM32\WINNSI.DLL:WINNSI.DLL (00007FFE2C0B0000), size: 45056 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2339
C:\Windows\system32\napinsp.dll:napinsp.dll (00007FFE25C50000), size: 90112 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\system32\pnrpnsp.dll:pnrpnsp.dll (00007FFE25C70000), size: 106496 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\system32\NLAapi.dll:NLAapi.dll (00007FFE2C790000), size: 98304 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.3808
C:\Windows\System32\winrnr.dll:winrnr.dll (00007FFE25C90000), size: 57344 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\rasadhlp.dll:rasadhlp.dll (00007FFE2A990000), size: 40960 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\System32\fwpuclnt.dll:fwpuclnt.dll (00007FFE2BF10000), size: 434176 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.0
C:\Windows\SYSTEM32\dbghelp.dll:dbghelp.dll (00007FFE1F9B0000), size: 1646592 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14321.1024
C:\Windows\SYSTEM32\SspiCli.dll:SspiCli.dll (00007FFE2E160000), size: 180224 (result: 0), SymType: '-deferred-', PDB: '', fileVersion: 10.0.14393.2580

========== OUTPUTTING STACK TRACE ==================

0x00007FFDD74234C4 (UnityPlayer) UnityMain
0x00000157980301D3 (Mono JIT Code) (wrapper managed-to-native) UnityEngine.Rendering.CommandBuffer:SetComputeBufferParam (UnityEngine.Rendering.CommandBuffer,UnityEngine.ComputeShader,int,int,UnityEngine.ComputeBuffer)
0x00000157A4054D3B (Mono JIT Code) UnityEngine.Rendering.HighDefinition.HDRenderPipeline:BuildCoarseStencilAndResolveIfNeeded (UnityEngine.Rendering.HighDefinition.HDCamera,UnityEngine.Rendering.RTHandle,UnityEngine.Rendering.RTHandle,UnityEngine.ComputeBuffer,UnityEngine.Rendering.CommandBuffer)
0x0000015776518F4B (Mono JIT Code) UnityEngine.Rendering.HighDefinition.HDRenderPipeline:ExecuteRenderRequest (UnityEngine.Rendering.HighDefinition.HDRenderPipeline/RenderRequest,UnityEngine.Rendering.ScriptableRenderContext,UnityEngine.Rendering.CommandBuffer,UnityEngine.Rendering.HighDefinition.AOVRequestData)
0x00000157764E66BB (Mono JIT Code) UnityEngine.Rendering.HighDefinition.HDRenderPipeline:Render (UnityEngine.Rendering.ScriptableRenderContext,UnityEngine.Camera[])
0x00000157764E1284 (Mono JIT Code) UnityEngine.Rendering.RenderPipeline:InternalRender (UnityEngine.Rendering.ScriptableRenderContext,UnityEngine.Camera[])
0x000001575411B5FB (Mono JIT Code) UnityEngine.Rendering.RenderPipelineManager:DoRenderLoop_Internal (UnityEngine.Rendering.RenderPipelineAsset,intptr)
0x000001575411B787 (Mono JIT Code) (wrapper runtime-invoke) <Module>:runtime_invoke_void_object_intptr (object,intptr,intptr,intptr)
0x00007FFDD568D6D0 (mono-2.0-bdwgc) mono_get_runtime_build_info
0x00007FFDD5612932 (mono-2.0-bdwgc) mono_perfcounters_init
0x00007FFDD561B98F (mono-2.0-bdwgc) mono_runtime_invoke
0x00007FFDD73C9A6D (UnityPlayer) UnityMain
0x00007FFDD73C6E23 (UnityPlayer) UnityMain
0x00007FFDD72330BE (UnityPlayer) UnityMain
0x00007FFDD70F5D77 (UnityPlayer) UnityMain
0x00007FFDD70F5678 (UnityPlayer) UnityMain
0x00007FFDD72996B7 (UnityPlayer) UnityMain
0x00007FFDD728B0E7 (UnityPlayer) UnityMain
0x00007FFDD728B183 (UnityPlayer) UnityMain
0x00007FFDD728D5BC (UnityPlayer) UnityMain
  ERROR: SymGetSymFromAddr64, GetLastError: 'Trying to access an invalid address.

' (Address: 00007FFDD704272E)
0x00007FFDD704272E (UnityPlayer) (function-name not available)
  ERROR: SymGetSymFromAddr64, GetLastError: 'Trying to access an invalid address.

' (Address: 00007FFDD704148A)
0x00007FFDD704148A (UnityPlayer) (function-name not available)
  ERROR: SymGetSymFromAddr64, GetLastError: Trying to access an invalid address.

' (Address: 00007FFDD704553C)
0x00007FFDD704553C (UnityPlayer) (function-name not available)
0x00007FFDD704902B (UnityPlayer) UnityMain
  ERROR: SymGetSymFromAddr64, GetLastError: 'Trying to access an invalid address.

' (Address: 00007FF75DB311F2)
0x00007FF75DB311F2 (simulator) (function-name not available)
0x00007FFE2F6784D4 (KERNEL32) BaseThreadInitThunk
0x00007FFE32051821 (ntdll) RtlUserThreadStart

========== END OF STACKTRACE ===========

A crash has been intercepted by the crash handler. For call stack and other details, see the latest crash report generated in:
 * C:/Users/lgsvl/AppData/Local/Temp/LGElectronics/SVLSimulator/Crashes
[CONN] Connected to WISE.

**Comment by EricBoiseLGSVL** [comment]
Thanks for logs.  That is a very odd bug.  Navigate to `C:/Users/user/AppData/LocalLow/LGElectronics/SVLSimulator` and delete the folder.  Then link simulator again to your account.

**Comment by tk-s05** [comment]
I tried but couldn't start the simulator.
I can launch the simulation in Unity, but I hope I can use the app.
Are there any other possible measures?

Hope for your help, thanks.

**Comment by EricBoiseLGSVL** [comment]
Sorry, we don't have many issues running simulator on windows.  Did you delete the appdata folder and run?  Can you please post the player.log as a txt file. it's difficult to parse inline.  Is this a server?  Do you have firewalls blocking?  Can you also check Nvidia control panel and make sure the application is forced to use the GPU?

**Comment by tk-s05** [comment]
I tried these...
・LGElectronics delete in appdata folder and run
・Unblocked all firewalls.
・I checked Nvidia control panel and make sure the application is forced to use the GPU

If there is no solution I may have to give up...Thanks

Player.log↓
[Player.log](https://github.com/lgsvl/simulator/files/6388456/Player.log)

**Comment by EricBoiseLGSVL** [comment]
Thanks for the log.  Comparing to my log you crash right when you try to connect to the WebUI.  Can you navigate to https://wise.svlsimulator.com ?  What browser are you using?

**Comment by EricBoiseLGSVL** [comment]
I feel this could be two things:
1) some firewall/setting is still blocking access to webui
2) You have a default app data folder like this C:\Users\YOURUSERNAME\AppData\LocalLow\LGElectronics\SVLSimulator.  I think your user name is not in English and/or we have a bug that can't write to it or is blocked.  Please try to edit the config file or add one `config.yml` then run again. Be sure to make the SVLDATA folder or you can change this absolute path to what you want, just keep it out of appdata and be sure nothing is blocking letting the app write to your drives.

```
headless: false
read_only: false
api_hostname: "localhost"
api_port: 8181
cloud_url: "https://wise.svlsimulator.com"
data_path: "C:\SVLDATA\LGElectronics\SVLSimulator"
```
